deepchem/splits/splitters.py

```diff
@@ -1211,79 +1425,51 @@ def update_distances(self, last_selected, cur_distances, distance_matrix,
         cur_distances[i] = new_dist
 
 
-class SpecifiedSplitter(Splitter):
```

Comment:
Good call getting rid of these classes! `IndiceSplitter` and `IndexSplitter` seem to provide this functionality already

Reply:
SpecifiedSplitter is still used in several examples and tutorials.  Those need to be fixed now.